gstreamer/src/pad.rs

```diff
@@ -882,11 +882,11 @@ unsafe extern "C" fn trampoline_pad_probe(
                 Some(PadProbeData::BufferList(from_glib_none(
                     data as *const ffi::GstBufferList,
                 )))
-            } else if (*data).type_ == Query::static_type().to_glib() {
-                data_type = Some(Query::static_type());
-                Some(PadProbeData::Query(QueryRef::from_mut_ptr(
-                    data as *mut ffi::GstQuery,
-                )))
+            } else if (*data).type_ == QueryRef::static_type().to_glib() {
+                data_type = Some(QueryRef::static_type());
+                Some(PadProbeData::Query(
+                    from_glib_full(data as *mut ffi::GstQuery)
```

Comment:
This is wrong: we only borrow the query here, we don't own it. And it must be writable